packages/beacon-node/test/e2e/api/impl/beacon/pool/endpoints.test.ts

```diff
@@ -0,0 +1,95 @@
+import {describe, beforeAll, afterAll, it, expect} from "vitest";
+import {createBeaconConfig} from "@lodestar/config";
+import {chainConfig as chainConfigDef} from "@lodestar/config/default";
+import {Api, getClient} from "@lodestar/api";
+import {ssz} from "@lodestar/types";
+import {LogLevel, testLogger} from "../../../../../utils/logger.js";
+import {getDevBeaconNode} from "../../../../../utils/node/beacon.js";
+import {BeaconNode} from "../../../../../../src/node/nodejs.js";
+
+describe("beacon pool api", function () {
+  const restPort = 9596;
+  const config = createBeaconConfig(chainConfigDef, Buffer.alloc(32, 0xaa));
+  const validatorCount = 512;
+
+  let bn: BeaconNode;
+  let client: Api["beacon"];
+
+  beforeAll(async () => {
+    bn = await getDevBeaconNode({
+      params: chainConfigDef,
+      options: {
+        sync: {isSingleNode: true},
+        network: {allowPublishToZeroPeers: true},
+        api: {
+          rest: {
+            enabled: true,
+            port: restPort,
+          },
+        },
+        chain: {blsVerifyAllMainThread: true},
+      },
+      validatorCount,
+      logger: testLogger("Node-A", {level: LogLevel.info}),
+    });
+    client = getClient({baseUrl: `http://127.0.0.1:${restPort}`}, {config}).beacon;
+  });
+
+  afterAll(async () => {
+    await bn.close();
+  });
+
+  describe("submitPoolAttestations", () => {
+    it("should return correctly formatted errors responses", async () => {
```

Comment:
could be more descriptive, maybe something like?
```suggestion
    it("should return an error with details about each failed attestation", async () => {
```